### PR TITLE
Marketing task

### PR DESCRIPTION
**Pull request by Lyla-Fischer**

### CODE REVIEW

File: tasks/2017/spread-the-word.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+call this file the submission file.
+
+### Star us on GitHub.
+
+There are four main repositories:
+[server/web](https://github.com/zulip/zulip),
+[mobile](https://github.com/zulip/zulip-mobile),
+[desktop](https://github.com/zulip/zulip-electron), and [Python
+API](https://github.com/zulip/python-zulip-api). For each repository, star the
+repository, and then write down the number of stars that repository has in
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: Maybe add something like "at the time of starring" or something to communicate that you don't have to keep updating this number.

File: scripts/2017/spread-the-word.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    max_instances = 30,
+    mentors = ['lylafisch@gmail.com'],
+    tags = ['marketing'], # free text
+    is_beginner = False,
+    # 1: Coding, 2: User Interface, 3: Documentation & Training,
+    # 4: Quality Assurance, 5: Outreach & Research
+    categories = [5],
+    time_to_complete_in_days = 3, # must be between 3 and 7
+    external_url = "https://github.com/zulip/zulip-gci/blob/master/tasks/2017/spread-the-word.md",
+    private_metadata = "intro-to-zulip-server",
```
Comment (lonerz) [inline scripts/2017/spread-the-word.py]: This doesn't seem right?

File: tasks/2017/spread-the-word.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+your submission file.
+
+### Share Zulip on StackShare
+
+1. Create an account at StackShare. Add your name and a picture, [i.e. do the
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: Is this supposed to be `[` or `(`?

File: tasks/2017/spread-the-word.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+work to have a real profile].
+
+2. Think of six technology products you really like, and consider adding Zulip
+to that list.
+
+3. For each product in 2: Add the product to your Stack, star it, and upvote at
+least three reasons why people like product that you find compelling.
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: like **the** product

File: tasks/2017/spread-the-word.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+4. Add a link to your profile to your submission file. The link should look like
+https://stackshare.io/Lyla-Fischer, but replace Lyla-Fischer with your
+username.
+
+### Write a review on the android play store
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: Android Play Store? I think it's Google Play Store and we should probably keep capitalization.

File: tasks/2017/spread-the-word.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Rate and review Zulip in the android play store. Five star reviews go a
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: r/android play store/Google Play Store

File: tasks/2017/spread-the-word.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+long way toward letting people know that Zulip is a quality product.
+
+2. Go to the Zulip app’s entry in the play store, scroll down to the “my
+review” section, and take a screenshot of your review and add that
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: section, take a screenshot of your review, and add that

File: tasks/2017/spread-the-word.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+screenshot to your submission file.
+
+### Write a review on the ios app store.
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: r/ios/iOS

File: tasks/2017/spread-the-word.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+## Background
+
+Upvotes and reviews make a big difference in the public perception of projects
+like Zulip.
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites.
+You're free to decide whether or not to do each individual task, but we do ask
+that you do at least five of the social media marketing tasks in order to get
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: If I counted correctly, there's only five tasks below. And I don't think many people can do both iOS and Android, so not sure if this `at least five` will work. Or maybe tasks are something else you had in mind?

Comment (Lyla-Fischer) [reply tasks/2017/spread-the-word.md]: Ah, no. That is left over from when there used to be more tasks. Thanks for doing this read-through. Things changed enough that there was a lot to catch.

File: tasks/2017/spread-the-word.md (hunk 3 above)
Comment (Lyla-Fischer) [reply tasks/2017/spread-the-word.md]: I'm not sure that it matters,  but I changed it to `(` just to be less weird.

**Comment by timabbott** [comment]
Merged after a bit of editing, thanks @Lyla-Fischer!

**Pull request closed without merging**